<?xml version="1.0"?>

<launch>
	
	<node name="keepout_map_server" pkg="keepout_map_server" type="keepout_map_server" ns="keepout" output="screen">
		<param name="map_width" value="992" />
		<param name="map_height" value="992" />
		<param name="map_resolution" value="0.01" />
		<param name="publish_interval" value="1" />
	</node>
	
</launch>
